### ISSUE TITLE
Is there a pretrained model we can use for inference?

### ISSUE THREAD

**Issue by dipikakhullar** [JiwanChung/tapm#21]
Hi @JiwanChung, 
Currently I am running the code with the following command:
```
python cli.py train with model=no_gt_sos fix_gpt_epoch=5 feature_names="['video', 'images']"```
```
I had some questions:
1. where will the model save/how long does it take to train?
2. I want to visualize the resulting model and use them to tweak the model a bit. Is there a pretrained model I can use? Does running this give the same results as in the paper? If not, how can I reproduce those results.

**Comment by JiwanChung** [comment]
1-1. The model weights will be saved under $REPO/data/ckpt .
1-2. The training time depends on the GPU used plus some hyperparameters. We used a single Tesla P40 for each experiment, which takes about a day (~24 hours) to converge.
2. Currently, we do not provide pretrained weights. However running the command would give similar results in the paper, except for a little performance fluctuation due to the random seed.

**Comment by JiwanChung** [comment]
Please reopen if you have further inquiries.
